### ISSUE TITLE
-Duse_system_wlroots=enabled fails to build with 2021-11-09 snapshot

### ISSUE THREAD

**Issue by jbeich** [WayfireWM/wayfire#1340]
Regressed by [wlroots@76bab68](https://gitlab.freedesktop.org/wlroots/wlroots/-/commit/76bab68e701b). From [error log](https://github.com/WayfireWM/wayfire/files/7523372/wayfire-0.7.2.159.log):
```c++
src/core/output-layout.cpp:668:14: error: use of undeclared identifier 'wlr_output_export_dmabuf'; did you mean 'wlr_output_event_damage'?
        if (!wlr_output_export_dmabuf(wo->handle, &attributes))
             ^
/usr/include/wlr/types/wlr_output.h:195:8: note: 'wlr_output_event_damage' declared here
struct wlr_output_event_damage {
       ^
```

**Comment by jbeich** [comment]
See also https://github.com/WayfireWM/wayfire-plugins-extra/issues/104

**Comment by jbeich** [comment]
Fixed by 0fc949ea2449
